universal-login-commons/src/core/constants/test.ts

```diff
@@ -25,6 +25,8 @@ export const TEST_GAS_PRICE = '1';
 
 export const TEST_GAS_LIMIT = 200000;
 
+export const TEST_TRANSACTION_OVERRIDES = {gasLimit: 1000000};
```

Comment:
Proposal: TEST_TXN_OVERRIDES_FOR_OVERRIDES or add comment with puprpose of this constant

Reply:
✅